index.js

```diff
@@ -1,149 +1,177 @@
-if (process.env.NODE_ENV !== 'production') {
-    require('dotenv').load();
+if (process.env.NODE_ENV !== "production") {
+  require("dotenv").config();
 }
 
-const path = require('path');
-const storage = require('azure-storage');
+const path = require("path");
+const { BlobServiceClient } = require("@azure/storage-blob");
 
-const blobService = storage.createBlobService();
+const connectionString = process.env.AZURE_STORAGE_CONNECTION_STRING;
+const blobServiceClient = BlobServiceClient.fromConnectionString(connectionString);
 
 const listContainers = async () => {
-    return new Promise((resolve, reject) => {
-        blobService.listContainersSegmented(null, (err, data) => {
-            if (err) {
-                reject(err);
-            } else {
-                resolve({ message: `${data.entries.length} containers`, containers: data.entries });
-            }
-        });
-    });
+  return new Promise(async (resolve, reject) => {
+    try {
+      const data = (await blobServiceClient.listContainers().byPage().next())

```

Comment:
This only listed one page of the containers. Do you also want to illustrate the usage of the continuation token?

Reply:
Ok, I have updated the code. Please have a look if the question is resolved.